_includes/about.html

```diff
@@ -1,20 +1,21 @@
-<section class="bg-primary" id="about">
+<section class="bg-dark" id="about">
     <div class="container">
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">It's hard work to collaborate on floral designs.</h2>
-                <hr class="light">
-                <p class="text-faded">A picture is worth a thousand words. But for custom flower orders, that picture won’t exist until the flower arrangement has already been made. Until then, clients and florists have to find other ways to communicate their artistic vision: emails, phone calls, Pinterest boards. It’s a process that takes trial and error, especially when a client doesn’t know what they want.
+                <h2 class="section-heading">
+                    At Foxtail, we know that the best custom bouquets come from local florist shops.</h2>
```

Comment:
```suggestion
                    At Foxtail, we know that the best custom bouquets come from local florists.</h2>
```

Reply:
I want to include the "shop" part here (unless it sounds awkward)